.github/workflows/bundle-release.yml

```diff
@@ -42,10 +42,10 @@ jobs:
           username: ${{ secrets.DOCKER_USER }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
-      - name: Set up GO 1.22.3
+      - name: Set up GO 1.22.4
```

Comment:
to fix cve:

![image](https://github.com/streamnative/function-mesh/assets/1858477/911badc5-11e9-44a2-baac-d119b7b71806)